tsfresh/transformers/feature_selector.py

```diff
@@ -82,7 +84,7 @@ def fit(self, X, y):
             y = pd.Series(y.copy())
 
         df_bh = check_fs_sig_bh(X, y, self.settings)
-        self.relevant_features = df_bh.loc[df_bh.rejected].Feature
+        self.relevant_features = df_bh.loc[df_bh.rejected].p_value
```

Comment:
For me, that is a public API change. (It will break some of my own scripts...)

Lets undo this changes and add the line 

`self.feature_importances_  = 1- df_bh.loc[df_bh.rejected].p_value`